pom.xml

```diff
@@ -90,9 +90,10 @@
         <dep.pmd.version>6.0.1</dep.pmd.version>
         <dep.slf4j.version>1.7.25</dep.slf4j.version>
         <dep.spring.version>4.2.4.RELEASE</dep.spring.version>
-        <findbugs.version>3.0.1</findbugs.version>
+        <findbugs.version>3.0.2</findbugs.version>
```

Comment:
should we just remove this, since we aren't using findbugs anymore?

Reply:
It's still being used for the `jsr305` artifact